docs/core/introduction.md

```diff
@@ -91,7 +91,7 @@ For more information, see [Releases and support for .NET Core and .NET 5](releas
 Here are some of the capabilities that .NET languages support:
 
 * [Type safety](../standard/base-types/common-type-system.md)
-* Type inference - [C#](../csharp/programming-guide/types/index.md#specifying-types-in-variable-declarations), [F#](../fsharp/language-reference/type-inference.md), [Visual Basic](../visual-basic/programming-guide/language-features/variables/local-type-inference.md)
+* Type inference - [C#](/dotnet/csharp/fundamentals/types#specifying-types-in-variable-declarations), [F#](../fsharp/language-reference/type-inference.md), [Visual Basic](../visual-basic/programming-guide/language-features/variables/local-type-inference.md)
```

Comment:
why is this link using the root?

Reply:
interesting. cleanrepo didn't find this one. Edited by hand.

Reply:
Hmm, maybe because it's an index file reference.

Reply:
maybe it should be `fundamentals/types/#bookmark`